.gitignore

```diff
@@ -1,6 +1,6 @@
 *.py[cod]
 fixtures/.*
-
+src
```

Comment:
The way I've got messytables installed at the moment is that it's in a
src/messytables directory off xypath. So I want to ignore its existence.

On Fri, Jan 23, 2015 at 11:47 AM, Francis Irving <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/scraperwiki/xypath/pull/62#discussion_r23445291>:
>
> > @@ -1,6 +1,6 @@
> >  *.py[cod]
> >  fixtures/.*
> > -
> > +src
>
> What's src/ for, out of interest?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/scraperwiki/xypath/pull/62/files#r23445291>.
>